packages-ts/gauntlet-terra-cw-plus/src/commands/multisig.ts

```diff
@@ -43,18 +43,34 @@ export const wrapCommand = (command) => {
 
       if (state.nextAction !== Action.CREATE) {
         this.require(
-          await this.isSameProposal(state.data, [this.toWasmMsg(message)]),
+          await this.isSameProposal(state.data, [this.toMsg(message)]),
           'The transaction generated is different from the proposal provided',
         )
       }
 
       return operations[state.nextAction](signer, Number(this.flags.proposal), message)
     }
 
-    isSameProposal = (proposalMsgs: WasmMsg[], generatedMsgs: WasmMsg[]) => {
+    isSameProposal = (proposalMsgs: (WasmMsg | BankMsg)[], generatedMsgs: (WasmMsg | BankMsg)[]) => {
       return isDeepEqual(proposalMsgs, generatedMsgs)
     }
 
+    toMsg = (message: MsgSend | MsgExecuteContract): BankMsg | WasmMsg => {
```

Comment:
Why do we need to do this 1:1 mapping? 
Why don't we just use `MsgSend | MsgExecuteContract` from the SDK directly?

Reply:
There is also `BankMsg` [defined in the SDK](https://github.com/terra-money/terra.js/blob/9e5f553de3ff3e975eaaf91b1f06e45658b1a5e0/src/core/bank/msgs/index.ts#L7) as `export type BankMsg = MsgSend | MsgMultiSend;`

Reply:
`BankMsg` and `WasmMsg` are actually multisig messages, maybe I should make them more clear

Reply:
`BankMsg` from the SDK contains the `MsgMultisend` which we don't want (for the moment)

Reply:
Ok, got it now. Thx!